### ISSUE TITLE
Say goodbye to menu button

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#245]
```
As described in 
http://android-developers.blogspot.com/2012/01/say-goodbye-to-menu-button.html 
we should add support for the Action Bar, which is a very nice Android 4.0 (API 
14) feature and full backwards compatible.

More information can be found at:
http://developer.android.com/design/patterns/actionbar.html
http://developer.android.com/guide/topics/ui/actionbar.html
```

Original issue reported on code.google.com by `fschm...@gmail.com` on 30 Jan 2012 at 11:15

**Comment by GoogleCodeExporter** [comment]
```
Working on a new user interface that will fix these issues
```

Original comment by `Florent....@gmail.com` on 20 May 2012 at 12:09
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 10 Jun 2012 at 3:58
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed with GTalkSMS v4.0 and released on market
```

Original comment by `Florent....@gmail.com` on 28 Jul 2012 at 4:20
- Changed state: **Deployed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed with GTalkSMS v4.0 and released on market
```

Original comment by `Florent....@gmail.com` on 28 Jul 2012 at 4:20
